front/app/containers/Admin/dashboard/components/ChartFilters/GroupFilter.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+// resources
+import GetGroups, { GetGroupsChildProps } from 'resources/GetGroups';
+
+// hooks
+import useLocalize, { Localize } from 'hooks/useLocalize';
+
+// components
+import { Box, Select } from '@citizenlab/cl2-component-library';
+
+// typings
+import { IOption } from 'typings';
+import { IGroupData } from 'services/groups';
+
+// i18n
+import { injectIntl, FormattedMessage } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+import messages from './messages';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+interface DataProps {
+  groups: GetGroupsChildProps;
+}
+
+interface InputProps {
+  currentGroupFilter?: string | null;
+  onGroupFilter: (filter: IOption) => void;
+}
+
+interface Props extends DataProps, InputProps {}
+
+const generateGroupOptions = (
+  groupsList: IGroupData[],
+  localize: Localize,
+  formatMessage: InjectedIntlProps['intl']['formatMessage']
```

Comment:
I'd change this to be more consistent with the way it's done above, like `intl: { formatMessage }`

Reply:
:+1: